config/initializers/rack_attack.rb

```diff
@@ -18,7 +18,9 @@ def remote_ip
     # whitelisting). It must implement .increment and .write like
     # ActiveSupport::Cache::Store
 
-    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(Redis.current)
+    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(
+      Redis.new(url: Figaro.env.sidekiq_redis_url)
```

Comment:
good point, I didn't notice that RA was using it until I setup redis on a remote host. I'll change it to `redis_url`.